flocker/node/agents/blockdevice_manager.py

```diff
@@ -0,0 +1,175 @@
+# -*- test-case-name: flocker.node.agents.test.test_blockdevice_manager -*-
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+This module implements interactions between the OS pertaining to block devices.
+This controls actions such as formatting and mounting a blockdevice.
+"""
+
+import psutil
+from subprocess import CalledProcessError, check_output, STDOUT
+
+from zope.interface import Interface, implementer
+
+from pyrsistent import PClass, field
+
+from twisted.python.filepath import FilePath
+
+from characteristic import attributes
+
+
+@attributes(["blockdevice", "mountpoint", "source_message"])
+class MountError(Exception):
+    """Exception class for errors in attempts to mount."""
```

Comment:
I'd write "Raised when we fail to mount something", because it's sort of obviously an exception class. I'm struggling to say that my way is actually _better_, so no change required there. :grey_question: